Src/Support/Google.Apis.Auth/OAuth2/ServiceAccountCredential.cs

```diff
@@ -210,6 +223,19 @@ public static ServiceAccountCredential FromServiceAccountData(Stream credentialD
             return result;
         }
 
+        /// <summary>
+        /// Constructs a new instance of the <see cref="ServiceAccountCredential"/> but with the
+        /// given <see cref="UseJwtAccessWithScopes"/> value.
+        /// </summary>
+        /// <param name="useJwtAccessWithScopes"></param>
```

Comment:
Sorry, missed this before: we need documentation for this parameter.

Reply:
oh thanks!